### PR TITLE
Add regex, xmldata and task BBEs

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

### CODE REVIEW

File: new-examples/index.json [hunk 1]
```diff
@@ -64,5 +64,46 @@
...
+    {
+        "title": "XML Data",
+        "column": 0,
+        "category": "Language concepts",
+        "samples": [
+            {
+                "name": "XML/JSON Conversion",
+                "url": "xml-json-conversion"
+            }
+        ]
```
Comment (daneshk) [inline new-examples/index.json:107]: Let's remove these entries. we don't need to update

File: new-examples/manage-scheduled-jobs/manage_scheduled_jobs.bal [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    isolated function init(int i, string jobIdentifier) {
+        self.i = i;
+        self.jobIdentifier = jobIdentifier;
+    }
+}
+
+public function main() returns error? {
+
+    //
+    check task:configureWorkerPool(6, 7000);
```
Comment (daneshk) [inline new-examples/manage-scheduled-jobs/manage_scheduled_jobs.bal:28]: Do we need this in the example. if not shall we remove it

File: new-examples/manage-scheduled-jobs/manage_scheduled_jobs.bal [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import ballerina/io;
+import ballerina/lang.runtime;
+import ballerina/task;
+import ballerina/time;
+
+// Creating a job to be executed by the scheduler.
```
Comment (daneshk) [inline new-examples/manage-scheduled-jobs/manage_scheduled_jobs.bal:6]:
```suggestion
// Creates a job to be executed by the scheduler.
```

File: new-examples/manage-scheduled-jobs/manage_scheduled_jobs.bal [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    }
+}
+
+public function main() returns error? {
+
+    // Gets the current time.
+    time:Utc currentUtc = time:utcNow();
+    // Increases the time by three seconds to set the starting delay for the scheduling job.
+    time:Utc newTime = time:utcAddSeconds(currentUtc, 5);
+    // Get the `time:Civil` for the given time.
```
Comment (daneshk) [inline new-examples/manage-scheduled-jobs/manage_scheduled_jobs.bal:31]:
```suggestion
    // Gets the `time:Civil` for the given time.
```

File: new-examples/task-frequency-job-execution/task_frequency_job_execution.bal [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    }
+}
+
+public function main() returns error? {
+
+    // Schedules the task to execute the job every second.
+    task:JobId id = check task:scheduleJobRecurByFrequency(new Job(0), 1);
+
+    // Waits for twelve seconds.
+    runtime:sleep(9);
```
Comment (daneshk) [inline new-examples/task-frequency-job-execution/task_frequency_job_execution.bal:28]: description and sleep seconds are different. is this intentional

File: new-examples/task-frequency-job-execution/task_frequency_job_execution.bal [hunk 6]
```diff
@@ -0,0 +1,32 @@
+import ballerina/io;
+import ballerina/lang.runtime;
+import ballerina/task;
+
+// Creating a job to be executed by the scheduler.
```
Comment (daneshk) [inline new-examples/task-frequency-job-execution/task_frequency_job_execution.bal:5]:
```suggestion
// Creates a job to be executed by the scheduler.
```

File: new-examples/task-one-time-job-execution/task_one_time_job_execution.bal [hunk 7]
```diff
@@ -0,0 +1,36 @@
+import ballerina/io;
+import ballerina/lang.runtime;
+import ballerina/task;
+import ballerina/time;
+
+// Creating a job to be executed by the scheduler.
```
Comment (daneshk) [inline new-examples/task-one-time-job-execution/task_one_time_job_execution.bal:6]:
```suggestion
// Creates a job to be executed by the scheduler.
```

**Review by daneshk — APPROVED** [review]
LGTM

File: new-examples/manage-scheduled-jobs/manage_scheduled_jobs.metatags [hunk 8]
```diff
@@ -0,0 +1,2 @@
+description: This BBE shows how to manage the scheduled jobs in Ballerina.
+keywords: ballerina, ballerina by example, bbe, task, job, scheduler
```
Comment (praneesha) [inline new-examples/manage-scheduled-jobs/manage_scheduled_jobs.metatags:2]:
```suggestion
keywords: ballerina, ballerina by example, BBE, task, job, scheduler
```
Please fix in all occurrences.

File: new-examples/regular-expressions/regular_expressions.bal [hunk 9]
```diff
@@ -0,0 +1,28 @@
+import ballerina/io;
+import ballerina/regex;
+
+public function main() {
+
+    // Checks whether the given string matches the provided regex.
+    // You need to provide the original `string` and the regex to
+    // which this string is to be matched.
```
Comment (praneesha) [inline new-examples/regular-expressions/regular_expressions.bal:8]:
```suggestion
    // this string, which is to be matched with.
```

File: new-examples/task-one-time-job-execution/task_one_time_job_execution.metatags [hunk 10]
```diff
@@ -0,0 +1,2 @@
+description: BBE on how to schedule the one-time job execution in Ballerina.
```
Comment (praneesha) [inline new-examples/task-one-time-job-execution/task_one_time_job_execution.metatags:1]:
```suggestion
description: BBE on how to schedule a one-time job execution in Ballerina.
```

File: new-examples/xml-json-conversion/xml_json_conversion.description [hunk 11]
```diff
@@ -0,0 +1,3 @@
+// The `xmldata:fromJson()` and `xmldata:toJson()` functions are using to do the conversions between JSON and XML.
```
Comment (praneesha) [inline new-examples/xml-json-conversion/xml_json_conversion.description:1]:
```suggestion
// The `xmldata:fromJson()` and `xmldata:toJson()` functions are used to do the conversions between JSON and XML.
```

**Review by praneesha — APPROVED** [review]

**Pull request merged**